footer.php

```diff
@@ -0,0 +1,17 @@
+<footer class="site-footer">
+			<div class="row">
+				<p>Copyright 2020 | Developer links: 
+					<a href="/job-board/index.php">Home</a>,
```

Comment:
these types of URLs assume that my project would be `localhost/job-board/` yes? what would happen if I change the URL?